configurations/codscript-tmLanguage.json

```diff
@@ -40,22 +40,22 @@
 		{
 			"comment": "Control Keywords",
 			"name": "markup.bold support.type.property-name",
-			"match": "\\b((?i)IF|THEN|ELSE|ELSE IF|ELSEIF|END IF|ENDIF|WHILE|END WHILE|ENDWHILE|FOR|TO|STEP|NEXT|DO|UNTIL|LOOP|SELECT|CASE|END SELECT|ENDSELECT|FUNCTION|RETURN|END FUNCTION|ENDFUNCTION)\\b"
+			"match": "\\b((?i)IF|THEN|ELSE|ELSE IF|ELSEIF|END IF|ENDIF|WHILE|END WHILE|ENDWHILE|FOR|TO|STEP|NEXT|DO|UNTIL|LOOP|LOOPUNTIL|SELECT|CASE|END SELECT|ENDSELECT|FUNCTION|RETURN|END FUNCTION|ENDFUNCTION)\\b"
 		},
 		{
 			"comment": "Special Keywords",
 			"name": "keyword.control",
-			"match": "(?<=^|\\s+)((?i)AND|OR|NOT|AS|IS|NEW|DIM|OBJECT|REQUIRES|INCLUDE|RUN)(?=\\s+)"
+			"match": "(?<=^|\\s+)((?i)AND|OR|NOT|AS|IS|NEW|DIM|OBJECT|REQUIRES|INCLUDE|RUN|BREAKPOINT)(?=\\s+)"
 		},
 		{
 			"comment": "Global Functions",
 			"name": "markup.bold",
-			"match": "\\b((?i)ACOS|ASIN|ATAN|COS|SIN|TAN|EXP|LOG|NUMBER|POW|SIGN|SQRT|SQR|ROUND|ROUNDDOWN|ROUNDUP|ASC|ASCII|CHR|GETFILEEXT|GETFILENAME|INSTR|LEFT|LEN|LOWER|LTRIM|MID|RIGHT|RTRIM|SUBSTRING|TRIM|UPPER|WILDCARD|DEBUG|EXEC|INPUTBOX|QUERY|ERROR)(?=\\()"
+			"match": "\\b((?i)ACOS|ASIN|ATAN|COS|SIN|TAN|EXP|LOG|NUMBER|POW|SIGN|SQRT|SQR|ROUND|ROUNDDOWN|ROUNDUP|ASC|ASCII|CHR|GETFILEEXT|GETFILENAME|INSTR|LEFT|LEN|LOWER|LTRIM|MID|RIGHT|RTRIM|SUBSTRING|TRIM|UPPER|WILDCARD|DEBUG|EXEC|INPUTBOX|QUERY|ERROR|OUTPUT)(?=\\()"
 		},
 		{
 			"comment": "Objects",
 			"name": "markup.bold string.regexp",
-			"match": "(^|\\s|=)((?i)ARRAY|FILE|FILELOCATOR|ITEMSTRUCT|ITEM|TASK)\\b"
```

Comment:
Your local version of [/configurations/codscript-tmLanguage.template.json](https://github.com/AgileBIM/FabCOD/blob/main/configurations/codscript-tmLanguage.template.json) is not up to date, copy/paste or download the contents of that link into your local version.

Reply:
Note to self: completed locally. Resolve after update committed.